deployments/loader/templates/configmap.yaml

```diff
@@ -33,9 +33,8 @@ data:
     {{- range .Values.baseModels }}
     - {{ . }}
     {{- end }}
-    models:
-    {{- range .Values.models }}
-    - {{ . }}
+    {{- if .Values.models}}
```

Comment:
should we use `with`?

Reply:
sorry for my bug!

probably better to have:

```
    {{- with .Values.models }}
    models:
    {{- toYaml . | nindent 4 }}
    {{- end }}
```

Reply:
`if` works too for this case?

Reply:
no strong opinion!